src/python_routing_v02/troute/routing/fast_reach/mc_reach.pyx

```diff
@@ -1226,7 +1239,7 @@ cpdef object compute_network_structured(
                 #Create compute reach kernel input buffer
                 for i in range(r.reach.mc_reach.num_segments):
                   segment = get_mc_segment(r, i)#r._segments[i]
-                  buf_view[i, 0] = qlat_array[ segment.id, <int>((timestep-1)/qlat_resample)]
+                  buf_view[i, 0] = qlat_array[ segment.id, <int>((timestep-1)/qts_subdivisions)]
```

Comment:
**BUG-FIX** - previously, the variable `qlat_resample` was causing an incorrect sampling of the qlateral data..

Reply:
As part of this bugfix, we should delete (or comment) line 1059